docs/release_notes/2024/August_2024/August_2024.md

```diff
@@ -9,31 +9,73 @@ tags:
   - august
 ---
 
+## 3.3.10.\* (August 30, 2024)
+
+- Prophecy Python libs version: 1.9.9
+- Prophecy Scala libs version: 8.0.31
+
+### Features {#Features3310}
+
+#### Spark Copilot Enhancements {#Spark3310}
+
+- **Source suggestion**: While working on your Pipeline, if there is an unconnected source, Data Copilot will now suggest other sources that you can add to it on the graph.
+
+#### SQL Copilot Enhancements {#SQL3310}
+
+- **SQL Subgraph**: We now support Subgraphs in your SQL models. Subgraphs allow you to wrap multiple different Gems under a single parent Gem. These logical groupings can simplify the Visual view of your model.
+
+- **Support for configurations in the Target Model**: The Target Model supports column selection and project-level configurations.
```

Comment:
I'm not seeing how this works on beta, so I'm not able to review this line item.